tasks/Summarize Discussion.mds

```diff
@@ -0,0 +1,51 @@
+> Specifically how time limits effect of ten and twenty minutes impact story generation (production), developing a plan of action (planning) 
+> or a summary of the discussion of an issue (discussion). The results are interpreted in terms of social entrainment, meaning how people modify and adjust behavior 
+> based on changes to condition. 
+
+
+# References
+> Hackman, J. R., Jones, L. E., & McGrath, J. E. (1967). A set of dimensions for describing the general properties of group-generated written passages. 
+Psychological Bulletin, 67(6), 379–390. https://doi.org/10.1037/h0024647 
+https://psycnet.apa.org/record/1967-10399-001
+
+> McGrath, J. (1985). Effects of time limits and task types on task performance and interaction of four-person groups. Journal of Personality and Social 
+> Psychology., 49(2), 395–407. https://doi.org/10.1037/0022-3514.49.2.395
+https://psycnet-apa-org.proxy.library.upenn.edu/fulltext/1986-01128-001.html
+
+
+
+# Stimuli
+## The visual components
+> There were no visual components to the study 
+
+## Materials for alternative versions of the experiment 
+> There were no alternative versions of these specific experiments however they did take place a part of a larger study 
+
+# Procedure
+## Steps
+> Step One: Participants told they were in study on task types and told time limits on the task performance group. They were told an observer
+> would be in the room to make periodic observations 
+> Step Two: The experimenter read the first problem to the group, give a time limit for the first session and instruct participants to use the full time available.
+      This instruction was intended to counteract the participants desire to finish the experiment as quickly as possible
+> Step Three: Once the first trial is complete, experimenters collected the solutions and pass out a questionnaire 
+> Step Four: Experimenter would read second problem when all participants had finished the questionnaire and would begin timing the session/making observations  
+> Step Five: Once the second trial was complete, experimenters collected the solutions and pass out a questionnaire
+> Step Six: once the questionnaires were completed the participants were debriefed and excused 
+
+## Roles 
+> No roles were assigned to participants however participants were assigned to four person same-sex groups for the tasks 
+
+
+# Criteria
+## Performance calculation
+> The quality of the written reports were assessed on four dimensions developed by Hackman, Jones and McGrath (1967) (https://psycnet.apa.org/record/1967-10399-001): 
+> The dimensions are: length, originality, quality of presentation, adequacy and creativity and one dimension assessing the length (Quantity)
+> All group products were rated on scales assessing five dimensions using a 7- pile sort-resort procedure by three judges- two assessed each dimension and all three 
```

Comment:
I found a more descriptive explanation of the "sort-resort" procedure here https://psycnet-apa-org.proxy.library.upenn.edu/fulltext/1967-10399-001.pdf if you wanted to go into more detail but I'm not sure that it's necessary

Reply:
I will add this to references, great find!